app/models/attribute/GlobalAttributeTable.scala

```diff
@@ -213,25 +228,40 @@ object GlobalAttributeTable {
-    } yield (
-      _ga.globalAttributeId, _lt.labelType, (_ga.lat, _ga.lng), _ga.severity, _ga.temporary, _ga.streetEdgeId,
-      _osm.osmWayId, _r.description, _l.labelId, (_lp.lat.get, _lp.lng.get), _l.gsvPanoramaId, _lp.heading, _lp.pitch,
-      _lp.zoom, (_lp.canvasX, _lp.canvasY), _lp.canvasWidth, _lp.canvasHeight, _l.agreeCount, _l.disagreeCount,
-      _l.notsureCount, _l.severity, _l.temporary
-    )
-
-    attributesWithLabels.list.map(GlobalAttributeWithLabelForAPI.tupled)
+    val attributesWithLabels = Q.queryNA[GlobalAttributeWithLabelForAPI](
+          s"""SELECT global_attribute.global_attribute_id, label_type.label_type, global_attribute.lat,
+          |        global_attribute.lng, global_attribute.severity, global_attribute.temporary,
+          |        global_attribute.street_edge_id, osm_way_street_edge.osm_way_id, region.description, label.label_id,
+          |        label_point.lat, label_point.lng, label.gsv_panorama_id, label_point.heading,
+          |        label_point.pitch, label_point.zoom, label_point.canvas_x, label_point.canvas_y,
+          |        label_point.canvas_width, label_point.canvas_height, label.agree_count, label.disagree_count,
+          |        label.notsure_count, label.severity, label.temporary, the_tags.tag_list, label.description
+          |FROM global_attribute
+          |INNER JOIN label_type ON global_attribute.label_type_id = label_type.label_type_id
+          |INNER JOIN region ON global_attribute.region_id = region.region_id
+          |INNER JOIN global_attribute_user_attribute ON global_attribute.global_attribute_id = global_attribute_user_attribute.global_attribute_id
+          |INNER JOIN user_attribute_label ON global_attribute_user_attribute.user_attribute_id = user_attribute_label.user_attribute_id
+          |INNER JOIN label ON user_attribute_label.label_id = label.label_id
+          |INNER JOIN label_point ON label.label_id = label_point.label_id
+          |INNER JOIN osm_way_street_edge ON global_attribute.street_edge_id = osm_way_street_edge.street_edge_id
+          |LEFT JOIN (
+          |    -- Puts set of tag_ids associated with the label in a comma-separated list in a string.
+          |    SELECT label_id, array_to_string(array_agg(tag.tag), ',') AS tag_list
+          |    FROM label_tag
+          |    INNER JOIN tag ON label_tag.tag_id = tag.tag_id
+          |    GROUP BY label_id
+          |) the_tags ON label.label_id = the_tags.label_id
+          |WHERE label_type.label_type <> 'Problem'
+          |    AND global_attribute.lat > $minLat
+          |    AND global_attribute.lat < $maxLat
+          |    AND global_attribute.lng > $minLng
+          |    AND global_attribute.lng < $maxLng
+          |    AND (global_attribute.severity IS NOT NULL
+          |    AND ${severity.getOrElse("") == "none"}
+          |    OR ${severity.isEmpty}
+          |    OR global_attribute.severity = ${toInt(severity).get})""".stripMargin
```

Comment:
Note that "none" is a valid input for severity so that we can specifically get labels with no severity. When doing that, we get a `None.get` error here.

Reply:
Ah I didn't catch that. I changed the call to `get` on line 262 to `getOrElse(-1)`, which resolves the error.